static/js/util/ProfileTab.js

```diff
@@ -4,20 +4,44 @@ import {
   boundDateField,
   boundTextField,
   boundSelectField,
+  editProfileObjectArray,
 } from './profile_edit';
 import COUNTRIES from '../countries';
+import LANGUAGE_CODES from '../language_codes';
 
 class ProfileTab extends React.Component {
   constructor(props) {
     super(props);
+
+    // bind our field methods to this
     this.boundTextField = boundTextField.bind(this);
     this.boundSelectField = boundSelectField.bind(this);
     this.boundDateField = boundDateField.bind(this);
+    this.editProfileObjectArray = editProfileObjectArray.bind(this);
+
+    // options we set (for select components)
     this.countryOptions = COUNTRIES.map(country => ({
       value: country.Code,
       label: country.Name
     }));
+    this.boolOptions = [
+      { value: true, label: 'Yes' },
```

Comment:
It looks like `true` and `false` are not actually working :open_mouth:  https://github.com/JedWatson/react-select/issues/546
Can you change it back?